packages/ts-codegen/src/__tests__/context.test.ts

```diff
@@ -2,10 +2,81 @@ import ts from "typescript";
 import { parse } from "graphql";
 import { blankGraphQLTag as graphql } from "../utilities";
 import { generateTS } from "..";
-import { ContextMap } from "../context";
+import { SubTypeNamespace } from "../codegen";
+import { OutputMetadata } from "../context/utilities";
 
 describe(generateTS, () => {
   describe("Tests basic syntax GraphQL syntax", () => {
+    const contextSubTypeMetadata = {
+      managers: {
+        user: {
+          name: "user",
```

Comment:
why this config field is needed? from what I see in the tests in always matches the `key` one level above it.

Reply:
that's format we agreed on with CDL